app/models/health/qualifying_activity.rb

```diff
@@ -395,11 +395,26 @@ def outreach?
     end
 
     def modifiers
+      TodoOrDie('Remove MH COVID flexibility', by: '2023-01-01')
+
       modifiers = []
+      case activity.to_sym
+      when :cha, :discharge_follow_up
+        if [:phone_call, :video_call].include?(mode_of_contact.to_sym)
+          contact_modifier = self.class.modes_of_contact[:in_person].try(:[], :code)
```

Comment:
Why is this a `try`?  It doesn't look like anything here should be changing, and it's referencing a class method, which should be relatively fixed right?

Reply:
Dead code left over from copy-paste :-)